diff --git a/node_modules/react-contexify/dist/react-contexify.esm.js b/node_modules/react-contexify/dist/react-contexify.esm.js
index 5a371f7..90b5321 100644
--- a/node_modules/react-contexify/dist/react-contexify.esm.js
+++ b/node_modules/react-contexify/dist/react-contexify.esm.js
@@ -1,4 +1,5 @@
 import React, { useContext, createContext, useRef, useEffect, Children, cloneElement, useReducer, useState } from 'react';
+import ReactDOM from 'react-dom';
 import cx from 'clsx';

 function _extends() {
@@ -520,12 +521,14 @@ var Menu = function Menu(_ref) {
   }

   function handleAnimationEnd() {
-    if (state.willLeave && state.visible) {
-      setState({
-        visible: false,
-        willLeave: false
-      });
-    }
+    ReactDOM.flushSync(() => {
+      if (state.willLeave && state.visible) {
+        setState({
+          visible: false,
+          willLeave: false
+        });
+      }
+    })
   }

   function computeAnimationClasses() {
